lib/reservation_monitor.py

```diff
@@ -100,14 +100,12 @@ def _check_flight_fares(self) -> None:
             try:
                 fare_checker.check_flight_price(flight)
```

Comment:
I think it would make more sense to have the successful healthcheck sent after this line instead of inside this function, as the failed healthchecks are also being called from this try/except.

Reply:
I can try to take a look at this today.

Reply:
Oh looks like I already did this.